### PR TITLE
New Pull request with fixed errors from before/ Added Google Api search bar

### PR DESCRIPTION
**Pull request by August0Rod**
Fixed a couple errors from the last pull request such as renaming the fragments to exercise instead of plural. Also implement the google maps search bar API that allows for search of places
Screen Shot 2022-07-05 at 3 52 40 PM

### CODE REVIEW

File: app/src/main/java/com/example/gymrats/fragments/ExerciseFragment.java [hunk 1]
```diff
@@ -77,7 +75,7 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         rvExercises = view.findViewById(R.id.rvExercises);
         exercises = new ArrayList<>();
 
-        ExercisesAdapter exercisesAdapter = new ExercisesAdapter(getContext(), exercises);
+        ExerciseAdapter exercisesAdapter = new ExerciseAdapter(getContext(), exercises);
```
Comment (ruiyangchen23) [inline app/src/main/java/com/example/gymrats/fragments/ExerciseFragment.java:78]: good catch!

File: app/src/main/java/com/example/gymrats/fragments/ExerciseFragment.java [hunk 2]
```diff
@@ -86,7 +84,7 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
 
         AsyncHttpClient client = new AsyncHttpClient();
 
-        client.get(EXERCISE_URL + "?limit=15" + "?category=" + categoryID + "&language=2", new JsonHttpResponseHandler() {
+        client.get(EXERCISE_URL + "?limit=200" + "&category=" + categoryID + "&language=2", new JsonHttpResponseHandler() {
```
Comment (ruiyangchen23) [inline app/src/main/java/com/example/gymrats/fragments/ExerciseFragment.java:87]: suggest non hardcode limit, but not a blocking issue

File: app/src/main/java/com/example/gymrats/fragments/MapFragment.java [hunk 3]
```diff
@@ -86,6 +96,29 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
...
+
+        SupportMapFragment mapFragment = SupportMapFragment.newInstance();
+
+        mapFragment.getMapAsync(this);
+
+        getChildFragmentManager()
+                .beginTransaction()
+                .add(R.id.map, mapFragment)
+                .commit();
+
```
Comment (ruiyangchen23) [inline app/src/main/java/com/example/gymrats/fragments/MapFragment.java:113]: format?

File: app/src/main/java/com/example/gymrats/fragments/ExerciseFragment.java (hunk 2 above)
Comment (August0Rod) [reply app/src/main/java/com/example/gymrats/fragments/ExerciseFragment.java:87]: > suggest non hardcode limit, but not a blocking issue



> suggest non hardcode limit, but not a blocking issue

going to fix this

**Pull request merged**